sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -263,41 +263,38 @@ aditof::Status CameraItof::initialize() {
                     m_modesVersion = 0;
                 }
             }
+        }
+        // Check weather new mixed modes are present, in case no, switch back to simple new modes
+        if (m_modesVersion == 2) //new mixed modes table is active
```

Comment:
I don't understand the comment. Why you say new mixed modes table is active?

Reply:
The comment remained there from previous implementations. Will remove it